soh/soh/z_message_OTR.cpp

```diff
@@ -187,4 +187,142 @@ extern "C" void OTRMessage_Init()
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_WARP_TRANSFER_LORE_3,
+        CustomMessage("I've taken to calling it a %bwarp&transfer%w. A fitting name, don't you&think? It's quite the convenience!",
+                    "Ich habe angefangen, es einen&%bWarp-Transfer%w zu nennen. Ein&passender Name, finden Sie nicht?&Das ist wirklich praktisch!",
+                    "J'ai pris l'habitude de l'appeler&%btransfert de distorsion%w. Un nom&approprié, vous ne trouvez pas?&C'est tout à fait pratique!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_FEE,
+        CustomMessage("Wow, over %y1000%w rupees saved! To&thank you, I'm dropping all&withdrawal and deposit fees!",
+                    "Wow, über %y1000%w Rupien gespart!&Als Dankeschön verzichte ich auf&alle Abhebungs- und&Einzahlungsgebühren!",
+                    "Wow, plus de %y1000%w roupies&économisées! Pour vous remercier,&je supprime tous les frais de&retrait et de dépôt!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_PIECE_OF_HEART,
+        CustomMessage("Astounding! A whopping %y5000%w&rupees saved! For such a feat, a&%rPiece of Heart%w is yours!",
+                    "Erstaunlich! Satte %y5000%w Rupien&gespart! Für eine solche Leistung&gehört Ihnen ein %rStück Herz%w!",
+                    "Étonnant! Un énorme %y5000%w roupies&économisées! Pour un tel exploit,&un %rFragment de Coeur%w!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_ZERO_AMOUNT,
+        CustomMessage(
+            "Eh, @? You can't&deposit the air in your pockets!&Come back when you've got some&real Rupees.",
+            "Äh, @? Sie können die&Luft nicht in Ihren Taschen&deponieren! Kommen Sie zurück,&wenn Sie echte Rupien haben.",
+            "Hein, @? Vous ne&pouvez pas déposer l'air dans vos&poches ! Revenez quand vous&aurez de vrais rubis."
+        ));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_INSUFFICIENT_BALANCE,
+        CustomMessage(
+            "Oh, @! It seems your&balance is running low. You need&more Rupees in the bank to make&this withdrawal.",
+            "Oh, @! Es scheint, dass&Ihr Guthaben zur Neige geht. Für&diese Auszahlung benötigen Sie&mehr Rupien auf der Bank.",
+            "Oh, @! Il semble que&votre solde soit faible. Vous avez&besoin de plus de roupies en&banque pour effectuer ce retrait."
+        ));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_WALLET_FULL,
+        CustomMessage(
+            "Whoa there, @! That&amount's too hefty for your wallet.&Trim it down a bit, eh?",
+            "Wow, @! Dieser Betrag&ist zu hoch für Ihren Geldbeutel.&Reduzieren Sie es ein wenig, oder?",
```

Comment:
The last sentence is in need of improvement.

`Reduzieren Sie es ein wenig, oder?` &#8594; `Reduzieren Sie ihn ein wenig, ja?`